### PR TITLE
Ros2 port

### PR DESCRIPTION
**Pull request by mikaelarguedas**
**To be merged on a ros2 dedicated branch**

Builds on top of https://github.com/ros-visualization/rqt_common_plugins/pull/455 and port the package to ROS 2 in https://github.com/ros-visualization/rqt_common_plugins/commit/e8bb09b89392affa6d05661d4e06cf28e901ecdd to address https://github.com/ros-visualization/rqt_common_plugins/issues/456

### CODE REVIEW

**Comment by ablasdel** [comment]
@dirk-thomas So merging this to master doesn't work right?
What is the default way for this? make a separate ros2 branch?

**Comment by mikaelarguedas** [comment]
> What is the default way for this? make a separate ros2 branch?

FWIW most ROS repositories releasing in both ROS1 and ROS2 have a `ros2` branch for ROS2 development and create some distro named branch when the content diverges from one distro to the next (e.g. `dashing` or `eloquent` branch)

File: package.xml [hunk 1]
```diff
@@ -30,31 +30,34 @@
...
+
+  <!-- Not yet ported to ROS 2
+  <exec_depend>rqt_bag</exec_depend>
+  <exec_depend>rqt_bag_plugins</exec_depend>
+  <exec_depend>rqt_dep</exec_depend>
+  <exec_depend>rqt_launch</exec_depend>
+  <exec_depend>rqt_logger_level</exec_depend>
+  <exec_depend>rqt_py_common</exec_depend>
   <exec_depend>rqt_web</exec_depend>
+  -->
```
Comment (dirk-thomas) [inline package.xml]: Instead of this separate block please comment out the existing lines if there is no ROS 2 port yet.

File: package.xml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="2">
   <name>rqt_common_plugins</name>
-  <version>0.4.8</version>
+  <version>1.0.0</version>
```
Comment (dirk-thomas) [inline package.xml]: Please don't update the version yet. That happens when `catkin_prepare_release` is being run.

File: package.xml [hunk 3]
```diff
@@ -30,31 +30,34 @@
...
   <exec_depend>rqt_console</exec_depend>
-  <exec_depend>rqt_dep</exec_depend>
   <exec_depend>rqt_graph</exec_depend>
   <exec_depend>rqt_image_view</exec_depend>
-  <exec_depend>rqt_launch</exec_depend>
-  <exec_depend>rqt_logger_level</exec_depend>
   <exec_depend>rqt_msg</exec_depend>
   <exec_depend>rqt_plot</exec_depend>
   <exec_depend>rqt_publisher</exec_depend>
-  <exec_depend>rqt_py_common</exec_depend>
```
Comment (dirk-thomas) [inline package.xml]: This package has been ported by now.

**Comment by mikaelarguedas** [comment]
Review comments addressed in https://github.com/ros-visualization/rqt_common_plugins/pull/457/commits/719e44063a0b6e92d65dbf67fc763ce2e539b1b1

**Comment by dirk-thomas** [comment]
Thanks for the contribution / update.

**Comment by dirk-thomas** [comment]
:ship: into Dashing and Eloquent

**Pull request merged**